CONTRIBUTING.md

```diff
@@ -403,6 +403,7 @@ Environment variables can be used to modify the behaviour of VSCode. The followi
 -   `AWS_TOOLKIT_AUTOMATION`: If tests are currently being ran
 -   `DEVELOPMENT_PATH`: The path to the aws toolkit vscode project
 -   `AWS_TOOLKIT_TEST_NO_COLOR`: If the tests should include colour in their output
+-   `E2E_TESTING_DIRECTORY` - Where the e2e tests should find the tests to run. If not specified use `src/testE2E`
```

Comment:
why is it specific to "e2e"? setting the test directory seems like a general concept. it would be clumsy to also add  `INTEG_TEST_DIRECTORY` later. 

also please use existing names as precedent. we already have  `TEST_FILE` https://github.com/aws/aws-toolkit-vscode/blob/961ff52e05ed45d25c749a2237de6b2b3237f076/.vscode/launch.json#L87

```suggestion
-   `TEST_DIR` - Directory where tests are found.
```